android/README.md

```diff
@@ -0,0 +1,220 @@
+# Introduction 
+
+This sample project covers:
+
+1. Setting up ImageKit Android SDK
+2. Rendering images
+3. Setting authentication context for the SDK
+4. Applying common image manipulations
+5. Adding overlays to images
+6. Client-side file uploading
+7. Adaptive video streaming
+8. Extensions for third party image loading libraries
+
+# How to run locally
+
+This project was bootstrapped with [Android Studio](https://developer.android.com/studio/install) using [Kotlin](https://kotlinlang.org/). Head over to the link for installation if not already done.
+
+## Setup authentication
+
+In `app/src/main/java/io/imagekit/imagekitdemo/MainActivity.kt`, set the following parameters for authentication:
+
+```kotlin
+ImageKit.init(
+   context = applicationContext,
+   publicKey = "YOUR_IMAGEKIT_PUBLIC_KEY",
+   urlEndpoint = "https://ik.imagekit.io/YOUR_IMAGEKIT_ID",
+   transformationPosition = TransformationPosition.PATH,
+)
+```
+
+Required parameters are `urlEndpoint`. The `publicKey` parameter is optional and only needed if you want to use the SDK for client-side file upload. 
+
+You can get the value of [URL-endpoint](https://imagekit.io/dashboard#url-endpoints) from your ImageKit dashboard.
+API public key can be obtained from the [developer](https://imagekit.io/dashboard#developers) section in your ImageKit dashboard.
+
+
+### Run the app in android emulator:
+
+Build and Run in Android Studio
+
+## Setup dummy backend for upload
+
+Move to the server directory
+```shell
+cd Server
+```
+
+Create `.env` file by copying `sample.env`
+
+```shell
+cp sample.env .env
+```
+
+Set the following keys in `.env`
+
+```shell
+PRIVATE_KEY=private_XXXXXXXXXXXX
+```
+
+API private key can be obtained from the [developer](https://imagekit.io/dashboard#developers) section in your ImageKit dashboard.
+
+Please note that file upload will only work if, along with the above, you have also defined `publicKey`, `urlEndpoint`, and `authenticationEndpoint` variables in `app/config/Imagekit.js`.
+
+Install packages:
+
+```bash
+npm install
+```
+
+## Run the Node.js server
+
+```
+npm start
+```
+
+Node server will run at `http://localhost:8080`.
+
+
+### Important Note
+For ease for loading file, `app/src/main/java/io/imagekit/imagekitdemo/FileUtils.kt` can be imported into your project and used like it is demonstrated in `app/src/main/java/io/imagekit/imagekitdemo/UploadFileActivity.kt`.
```

Comment:
Having it as a blockquote would increase it's visibility and reduce it's criticality, because it is something to ease the customers integration process but not exactly critical

Just a recommendation from my end, can skip this also

```suggestion
> #### Recommendation
> For ease for loading file, `app/src/main/java/io/imagekit/imagekitdemo/FileUtils.kt` can be imported into your project and used like it is demonstrated in `app/src/main/java/io/imagekit/imagekitdemo/UploadFileActivity.kt`.
```